src/app/shared/branding.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { Fabric8UIConfig } from '../shared/config/fabric8-ui-config';
+
+import openshiftLogo from '../../assets/images/OpenShift-io_logo.png';
+import fabric8Logo from '../../assets/images/fabric8_logo.png';
+
+@Injectable()
+export class BrandingService {
+  private isFabric8: boolean;
+
+  constructor(
+    private fabric8UIConfig: Fabric8UIConfig,
+  ) {
+    this.isFabric8 = this.fabric8UIConfig.branding && this.fabric8UIConfig.branding === "fabric8";
+    console.log('isFabric8', this.isFabric8);
+  }
+
+  get name(): string {
```

Comment:
This feels a bit odd to me that you aren't using a private variable for these Getter methods.